### ISSUE TITLE
As per new update by google We cannot use any 32 bit native library after 21 August 2019 This library is also 32 bit when i analyze my apk How to get it support for 64 bit architecture.

### ISSUE THREAD

**Issue by Mahive** [barteksc/AndroidPdfViewer#745]
Please help on the new update from google on using only 64 bit libraries after 21 august deadline this library is using 32 bit native code how to update my app to make it compatible as  per new updates.

**Comment by mikemike396** [comment]
So when I analyzed the APK it shows x86, x86_64, armeabi-v7a, arm64-v8a. From my understanding of Googles deadlines, they want all apps to support 64 bit. In this case, this library does support 64 and 32-bit devices. Google doesn't want apps to only support 32-bit after August 1, 2019. I believe this library is within Googles requirements. 

https://android-developers.googleblog.com/2019/01/get-your-apps-ready-for-64-bit.html

**Comment by rafescu** [comment]
I also analyzed my APK, like @mikemike396 mentioned, and got all those librarys (armeabi-v7a, x86,arm64-v8a and x86_64), so I believe this library is within Google requirements.

**Comment by rafescu** [comment]
![screen shot 2019-02-06 at 14 45 04](https://user-images.githubusercontent.com/18332454/52349159-d5bf1a00-2a1d-11e9-81f9-3796baca4580.png)
But I have a doubt, on my analysis, on the lib folder, I detect a mips and armeabi folder, which don't have a similar folder to 64 bits (x86->x86_64,armeabi-v7a->arm64-v8a). Do I need to worry? @mikemike396

**Comment by mikemike396** [comment]
@rafescu I don't believe mips has a 64bit version, and armeabi is just the version before v7a, and v8a. Both of those support only 32-bit devices, but the way I read Googles article including those libraries should not be an issue. Now if this library only had (mips, armeabi, and x86) then we would be in violation of there new requirments.

**Comment by rafescu** [comment]
@mikemike396 yep, I guess the same. Thanks!

**Comment by lorenzos** [comment]
Anyone got confirmation uploading an APK to the Play Store? Lately the console started showing explicit warnings if the APK won't be valid  in August.

Looks like the (native library has 64 bit support, here](https://github.com/barteksc/PdfiumAndroid/blob/master/CHANGELOG.md#190-2018-06-29).

**Comment by zauli83** [comment]
Stille no news here?

**Comment by Mahive** [comment]
it's supporting 64bit also bro

**Comment by bagusflyer** [comment]
I can't pass the version check. It always remind me the app is in 32 bits. Any idea?

**Comment by bagusflyer** [comment]
> Anyone got confirmation uploading an APK to the Play Store? Lately the console started showing explicit warnings if the APK won't be valid in August.
> 
> Looks like the [native library has 64 bit support, here](https://github.com/barteksc/PdfiumAndroid/blob/master/CHANGELOG.md#190-2018-06-29).

I have problem to upload the `aab` to play store. Any idea?
